src/scripts/main.js

```diff
@@ -1,7 +1,43 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const block = document.createElement('div');
+  const h2 = document.createElement('h2');
+  const paragraph = document.createElement('p');
+
+  block.classList.add('notification');
+  h2.classList.add('title');
+  h2.style.fontSize = '16px';
+  block.style.top = `${posTop}px`;
+  block.style.right = `${posRight}px`;
+  block.append(h2);
+  block.append(paragraph);
+
+  h2.innerText = title;
+  paragraph.innerText = description;
+
+  switch (type) {
+    case 'success':
+      block.classList.add('success');
+      break;
+
+    case 'error':
+      block.classList.add('error');
+      break;
+
+    case 'warning':
+      block.classList.add('warning');
+      break;
+
+    default:
+      break;
+  }
```

Comment:
no needs to use this switch statement
```suggestion
```

Reply:
Done. I do understand, that this is useless here, but it seems, that my love for switches made me mad)).